src/npm.ts

```diff
@@ -1,23 +1,85 @@
-import util from 'util';
-import npm from 'npm';
+import { spawn } from 'child_process';
+import { lstat, rename, rm, writeFile } from 'fs/promises';
+import { IHasNodePath } from './types';
+
+const temporarilyMovedFiles: {[key: string]: string} = {
+  'package.json': `.package.json-${process.pid}`,
+  'package-lock.json': `.package-lock.json-${process.pid}`,
+};
 
 export default class NPM {
-  public static async install (...args: string[]) {
-    const npmInstall = util.promisify(npm.commands.install);
-    await npmInstall(args);
+  static async removePackageJson () {
```

Comment:
Makes the intent clearer. `remove` is more synonymous with `delete`.

```suggestion
  static async renamePackageJson () {
```